tests/community.test.js

```diff
@@ -0,0 +1,81 @@
+const request = require('supertest');
+
+describe('API Routes', () => {
+  const app = request('http://localhost:3000');
```

Comment:
Jus double checking, I thought Grace was going to be port 3000?

Reply:
Nah, I'm 3000/3001